indexing-service/src/main/java/org/apache/druid/indexing/common/TaskStorageDirTracker.java

```diff
@@ -189,6 +196,7 @@ public synchronized Map<String, StorageSlot> findExistingTaskDirs(List<String> t
         missingIds.add(taskId);
       } else {
         candidateSlot.runningTaskId = taskId;
+        ++numUsedSlots;
```

Comment:
Please put this before the above line for symmetry with the other place where this is being incremented.